ru.sview.sView.json

```diff
@@ -35,8 +35,8 @@
             "sources" : [
                 {
                     "type": "archive",
-                    "url" : "https://hyperrealm.github.io/libconfig/dist/libconfig-1.7.2.tar.gz",
-                    "sha256": "7c3c7a9c73ff3302084386e96f903eb62ce06953bb1666235fac74363a16fad9"
+                    "url" : "https://hyperrealm.github.io/libconfig/dist/libconfig-1.7.3.tar.gz",
+                    "sha256": "545166d6cac037744381d1e9cc5a5405094e7bfad16a411699bcff40bbb31ee7"
```

Comment:
I've cherry-picked that change in https://github.com/flathub/ru.sview.sView/pull/4